workspaces/playlist/plugins/playlist/src/components/EntityPlaylistDialog/EntityPlaylistDialog.tsx

```diff
@@ -256,10 +257,12 @@ export const EntityPlaylistDialog = (props: EntityPlaylistDialogProps) => {
                     >
                       <ListItemText
                         primary={list.name}
-                        secondary={`by ${humanizeEntityRef(
-                          parseEntityRef(list.owner),
-                          { defaultKind: 'group' },
-                        )} · ${list.entities} entities ${
+                        secondary={`by ${
+                          entityPresentationApi.forEntity(
+                            stringifyEntityRef(parseEntityRef(list.owner)),
+                            { defaultKind: 'group' },
+                          ).snapshot.primaryTitle
+                        } · ${list.entities} entities ${
                           !list.public ? '· Private' : ''
                         }`}
```

Comment:
Instead of using `entityPresentationApi` directly (which you probably don't want to do due to reasons [described here](https://github.com/backstage/backstage/pull/24081#discussion_r1557386266)), I think you should be able to use `EntityDisplayName` to render like so:
```tsx
                        secondary={
                          <Typography
                            variant="body2"
                            color="textSecondary"
                          >
                            by{' '}
                            <EntityDisplayName 
                              hideIcon 
                              disableTooltip 
                              defaultKind="group"
                              entityRef={parseEntityRef(list.owner)} 
                            />
                            {' · '}{list.entities} entities {!list.public ? '· Private' : ''}
                          </Typography> 
                        }
```

You might have to adjust the `Typography` variant to match the current styling - haven't tested this myself

Reply:
Sure will do this changes.